scripts/ci/check_compliance.py

```diff
@@ -204,16 +204,29 @@ def run(self):
         if not os.path.exists(checkpatch):
             self.skip(f'{checkpatch} not found')
 
+        # check for Perl installation on Windows
+        if os.name == 'nt':
+            if not shutil.which('perl'):
+                self.failure("Perl not installed - required for checkpatch.pl. Please install Strawberry Perl or add to PATH.")
+                return
+            else:
+                cmd = ['perl',checkpatch]
```

Comment:
Nit - spaces between the cmd tokens.
```suggestion
                cmd = ['perl', checkpatch]
```

Reply:
Amended commit to add this space between command tokens.